scripts/utils/tagger.ts

```diff
@@ -359,6 +395,28 @@ async function tagPyrites() {
     },
   );
   console.timeLog("etl", `Finished tagging pyrites`);
+
+  console.timeLog("etl", "Reporting new golds to OAF webhook");
+  for (const [category, run] of Object.entries(await getGolds())) {
```

Comment:
Quick check-in: will this do bad async things or will this work the way you want?

Reply:
I assume because the function is async it's fine but I could imagine this being a dumb thing that doesn't work the way you want.

Reply:
It works the way we want